extension/api.js

```diff
@@ -1,9 +1,12 @@
-const { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
-const { XPCOMUtils } = ChromeUtils.import(
-  "resource://gre/modules/XPCOMUtils.jsm"
-);
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/* globals ExtensionAPI, Services, XPCOMUtils, WebExtensionPolicy */
+
 XPCOMUtils.defineLazyGlobalGetters(this, ["ChannelWrapper"]);
-const { WebExtensionPolicy } = Cu.getGlobalForObject(Services);
```

Comment:
This cannot be removed if the system add-on targets Firefox < 91, because the symbol was added to the experimental namespace in 91, as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1716642 (https://hg.mozilla.org/mozilla-central/rev/f020a73949bf#l8.2)